src/codewhisperer/commands/basicCommands.ts

```diff
@@ -57,14 +56,6 @@ export const enableCodeSuggestions = Commands.declare(
         await set(CodeWhispererConstants.autoTriggerEnabledKey, true, context.extensionContext.globalState)
         await vscode.commands.executeCommand('setContext', 'CODEWHISPERER_ENABLED', true)
         await vscode.commands.executeCommand('aws.codeWhisperer.refresh')
-        const prompts = PromptSettings.instance
-
-        const shouldShow = await prompts.isPromptEnabled('codeWhispererNewWelcomeMessage')
-        //If user login old or new, If welcome message is not shown then open the Getting Started Page after this mark it as SHOWN.
-        if (shouldShow) {
-            vscode.commands.executeCommand('aws.codeWhisperer.gettingStarted', createGettingStartedNode())
```

Comment:
why are we removing `createGettingStartedNode` ? is there other places that this node gets created?

Reply:
I think we do not need to declare which callback function to execute,
```
export const createGettingStartedNode = () =>
    CodeWhispererCommandDeclarations.instance.declared.showGettingStartedPage
        .build('codewhispererDeveloperTools')
        .asTreeNode({
            label: localize('AWS.explorerNode.codewhispererGettingStartedNode.label', 'Learn'),
            iconPath: getIcon('aws-codewhisperer-learn'),
        })
```

instead using this : 
[code](https://github.com/LaxmanReddyAileni/aws-toolkit-vscode/blob/feature/gettingstarted/src/codewhisperer/commands/gettingStartedPageCommands.ts)

```
     showGettingStartedPage: Commands.from(CodeWhispererCommandBackend).declareShowGettingStartedPage(
            'aws.codeWhisperer.gettingStarted'
        ),
```

Correct me if I am wrong!
I tried to generate package and it works in all test cases.

Reply:
I guess what's confusing here to me is:
here we want to show the page, what does it have to do with `createGettingStartedNode()` at all if `createGettingStartedNode()` is meant to just render to `Learn` node in the devTools panel?

Reply:
Yeah thats the reason I removed it from the existing code.